package.json

```diff
@@ -18,6 +18,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-native-fontawesome": "^0.2.7",
     "@hathor/wallet-lib": "^1.0.1",
+    "@jest/globals": "^29.7.0",
```

Comment:
Oh... this will go to the dev after we rebase.

Reply:
But this PR (https://github.com/HathorNetwork/hathor-wallet-mobile/pull/332) does not update the package lock, only this one. We should update the package lock only after this rebase. Can you do it before merging?

Reply:
Oh! It don't change now because I created another PR only to update the package-lock.json, see https://github.com/HathorNetwork/hathor-wallet-mobile/pull/335

Reply:
It is resolved with the rebase.